tests/test_main.py

```diff
@@ -15,3 +15,14 @@ def test_when_calling_main_without_arguments_then_usage(self):
         with self.assertRaisesRegex(Exception,
                                     'USAGE: %s results.tap' % __file__):
             main([__file__])
+
+    @unittest.expectedFailure
+    def test_produce_traceback_output(self):
+        """Add traceback on TAP ouput"""
+        main([__file__])
+
+    def test_parse_file_with_traceback(self):
+        """Parse file with traceback"""
+        with patch('tcms_tap_plugin.Plugin.parse') as parse:
+            main([__file__, 'traceback.taptest'])
+            parse.assert_called_with('traceback.taptest')
```

Comment:
What is the intention of this test? It opens the program with the provided tap file and then asserts that we've opened the program with the file name as argument. This doesn't sound like a meaningful test.

Which will also render the `.taptest` file obsolete.

Instead what it should test is that the traceback was recorded and passed down to the test case.